content/module-reference/utility-modules/shared/scopes.md

```diff
@@ -111,15 +111,21 @@ Bear in mind that these controls are provided as a basic guide to achieving colo
 
 # exposure adjustment
 
-The histogram, waveform, and RGB parade scopes can be used to directly alter the _exposure_ and _black level_ of the [_exposure_](../../processing-modules/exposure.md) module.
+The histogram, waveform, and RGB parade scopes can be used to directly alter the exposure and black level of the exposure module.
```

Comment:
In other places in the docs, individual controls (like _exposure_ and _black level_ here) are highlighted using _italics_.
Also, why remove the link to the exposure module's docs?

Reply:
Thanks for looking at this @kofa73. This is my first attempt at learning how to help with documentation. I just copied what was already in the user manual and added the bare minimum to update the documentation to be correct. I didn't mean to remove any links or change the use of italics, but I see the inconsistency in the use of italics in the document I copied from. Can you advice how I retain the link? I am still on my L plates here.

Reply:
The site uses markdown format, which is pretty simple, and the same as github uses for its comments. Github provides a guide ([here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)) or there are various "cheat sheets" available (for example [here](https://www.markdownguide.org/cheat-sheet/))

Reply:
Should I rewrite my correction or is this close enough to be edited and used?

Reply:
Just restore the original text:
```suggestion
The histogram, waveform, and RGB parade scopes can be used to directly alter the _exposure_ and _black level_ of the [_exposure_](../../processing-modules/exposure.md) module.
```

Reply:
If this PR is a one-off I don't mind making edits when I get around to merging, but if you intend to submit more pull requests in the future it would be good to learn the process for making changes.